tests/rbd_mirror/test_mirror_move_secondary_trash.py

```diff
@@ -0,0 +1,77 @@
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror.rbd_mirror_utils import rbd_mirror_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_image_move_secondary_trash(rbd_mirror, pool_type, **kw):
```

Comment:
Method doc string is missing.

Reply:
Actually i have added the doc string in run method , but i will redesign the method doc and update.

Reply:
Done.